### PR TITLE
Update installing-stencil.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3118](https://jira.bigcommerce.com/browse/DEVDOCS-3118)

## What changed?
Updated Mac's installing stencil cli instructions
My computer does not have an M1 chip, it has an Intel Core i7 chip so I am not able to validate the steps. I was able to research and find information [here](https://github.com/bigcommerce/stencil-cli/issues/681) and confirmed with Jordan Arldt in [slack](https://bigcommerce.slack.com/archives/C011D5EBA57/p1631818833038500).

### CODE REVIEW

**Comment by traci707** [comment]
My computer does not have a M1 chip, it has an Intel Core i7 chip so I am not able to validate the steps.  I was able to research and find information [here](https://github.com/bigcommerce/stencil-cli/issues/681) and confirmed with Jordan Arldt in [slack](https://bigcommerce.slack.com/archives/C011D5EBA57/p1631818833038500).

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 1]
```diff
@@ -17,7 +17,19 @@ This article contains the detailed instructions needed to install and configure
 
 ## Installing on Mac
 
-To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands.
+To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands. Refer to [Stencil CLI README.MD](https://github.com/bigcommerce/stencil-cli) for latest `node` version supported.
```
Comment (ektravel) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:20]:
```suggestion
To install Stencil CLI and its dependencies on Mac, open a terminal and run the following commands. Refer to [Stencil CLI README.MD](https://github.com/bigcommerce/stencil-cli) for latest `node` version supported.

```
Changed contraction to possessive pronoun.

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 2]
```diff
@@ -35,21 +47,33 @@ nvm use 12
...
 
+```shell
+
+# Install Node Version Manager (nvm)
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
+
+# Install Stencil CLI supported version of Node.js
+nvm install 12.1
+
+# Switch to Stencil CLI supported version of Node.js:
```
Comment (ektravel) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:70]:
```suggestion
# Switch to Stencil CLI supported version of Node.js

```

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 3]
```diff
@@ -35,21 +47,33 @@ nvm use 12
...
+nvm use 12.1
+
+# Install Stencil CLI
+npm install -g @bigcommerce/stencil-cli
+```
+
 ## Installing on Windows
 There's two methods for installing Stencil CLI and its dependencies on Windows.
```
Comment (ektravel) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:78]:
```suggestion
There are two methods for installing Stencil CLI and its dependencies on Windows.

```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**